lib/content_scripts/website/pages/utils.css

```diff
@@ -0,0 +1,17 @@
+.anime-list-link-container {
+    display: flex;
+    align-items: center;
+    gap: .25rem;
```

Comment:
While a little more gap to align with the watchlist icon sounds good, it's not a requirement for this task.

Reply:
The added the gap primary to have a little spacing between the icons if both MAL and AniList are activated. I've fixed this now by adding `margin-right: .25rem` to all anime list buttons except the last one.